blackbox_repository/tabulated_benchmark.py

```diff
@@ -130,12 +130,15 @@ def __getstate__(self):
             'blackbox_name': self.blackbox_name,
             'dataset': self.dataset,
             'surrogate': self._surrogate,
+            'elapsed_time_attr': self.elapsed_time_attr,
         }
 
     def __setstate__(self, state):
+        super().__init__(elapsed_time_attr=state['elapsed_time_attr'])
```

Comment:
Thanks, this would work, but could you also add simulator_config and tuner_sleep_time, which are two other members of SimulatorBackend we would want to get and set. Then, this looks good.

Reply:
Can we do this in another PR? Setting `simulator_config`, `tuner_sleep_time` is currently not done in `_BlackboxSimulatorBackend` (the default are used). I do not mind if we allow setting them in BlackboxSimulatorBackend but this would be a separate change.

Reply:
That is not quite true, they are set via **simulatorbackend_kwargs. And this is used by launch_hpo.py, so not doing it would break for example the setting of tuner_sleep_time.

It is just two more lines. Thanks.

BTW: Not in the PR, but would it not also work to call super().__setstate__, super().__getstate__ in the right way? I just don't know how state is organized then. Maybe it is just one flat dict ...

In any case, thanks for this overall change, because it certainly avoids huge Dill files when serializing the Tuner, so we really want to have it.

Reply:
> That is not quite true, they are set via **simulatorbackend_kwargs. And this is used by launch_hpo.py, so not doing it would break for example the setting of tuner_sleep_time.

To be clear I do not mind doing calling `simulatorbackend_kwargs` but this was not in "main" when the PR nor this comment was sent. It was just merged with a recent PR:
 https://github.com/awslabs/syne-tune/blob/ba0b25431143c26865d28ea403f6c0b27a10e20b/blackbox_repository/tabulated_benchmark.py#L32

Now that the PR is merged, I can update with this change.

Reply:
Ah you are right!